static/scss/button.scss

```diff
@@ -14,10 +14,20 @@
   padding: 6px 16px;
 }
 
-.green-button {
+.green-button,
+.green-button-outline:hover {
   background-color: $studio-green;
   color: white;
   font-size: 14px;
   border-radius: 8px;
   padding: 6px 16px;
 }
+
+.green-button-outline {
+  border-color: $studio-green;
+  background-color: white;
+  color: $studio-green;
+  font-size: 14px;
+  border-radius: 8px;
+  padding: 6px 16px;
+}
```

Comment:
There are some styles being repeated here from the `.green-button` definition. Can you combine these?

```scss
.green-button, .green-button-outline {
  font-size: 14px;
  border-radius: 8px;
  padding: 6px 16px;
}

.green-button,
.green-button-outline:hover {
  background-color: $studio-green;
  color: white;
}

.green-button-outline {
  border-color: $studio-green;
  background-color: white;
  color: $studio-green;
}
```